conf/test_v2.config

```diff
@@ -0,0 +1,49 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/pixelator -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+
+aws.client.downloadParallel = true
+
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    // input          = params.pipelines_testdata_base_path + 'pixelator/samplesheet/samplesheet_v2.csv'
+    // input_basedir  = params.pipelines_testdata_base_path + 'pixelator/testdata/'
+    input          = params.pipelines_testdata_base_path + 'samplesheet/samplesheet_v2.csv'
+    input_basedir  = params.pipelines_testdata_base_path + 'testdata/'
+
+    multiplet_recovery = true
+    min_size = 2
+    max_size = 100000
+    compute_polarization = true
+    use_full_bipartite = true
+    colocalization_min_region_count = 0
+    colocalization_n_permutations = 10
+    colocalization_neighbourhood_size = 1
+
+    // Number of pivots is larger than the number of components in the graph for this small testdata
+    compute_svd_var_explained = false
```

Comment:
I think this parameter is not used here.

Reply:
Right, thanks!